### PR TITLE
Dev refactor

### PR DESCRIPTION
**Pull request by patiem**

### CODE REVIEW

File: client/src/main/java/gui/playing/PlayBoardController.java [hunk 1]
```diff
@@ -59,7 +61,7 @@
   private Sea sea;
   private LanguageVersion languageVersion = new LanguageVersion();
   private OutputChannelDispatcher outputChannelDispatcher = new OutputChannelDispatcher();
-
+  private Map<String,String> messageMap = new HashMap<>();
```
Comment (rivalez) [inline client/src/main/java/gui/playing/PlayBoardController.java]: what about Enum, String map?

File: client/src/main/java/model/Fleet.java [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+import gui.fields.Field;
+
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class Fleet {
+
+public class Fleet implements Iterable<Ship>{
```
Comment (rivalez) [inline client/src/main/java/model/Fleet.java]: do you really need implement Iterable?

File: client/src/main/java/model/RandomFleetGenerator.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  private void createFleetWithEmptyShips() {
+    List<Integer> shipLengths = Arrays.asList(4, 3, 3, 2, 2, 2, 1, 1, 1, 1);
+    for (Integer length : shipLengths) {
+      randomFleet.addShip(new Ship(length));
+    }
+  }
+
+  private void buildShipsInFleet() {
+    for (Ship ship : randomFleet) {
+      new ShipBuilder(ship).build();
```
Comment (rivalez) [inline client/src/main/java/model/RandomFleetGenerator.java]: ????? what that isn't builder

File: client/src/main/java/model/ShipBoundariesPositions.java [hunk 4]
```diff
@@ -22,17 +20,12 @@ public ShipBoundariesPositions(Sea sea) {
...
 
-  public ShipBoundariesPositions calculateShipBoundariesPositions(Ship ship) {
-
-    for (Field mast : ship) {
-      boundariesForMast(mast.position());
-    }
-    boundaries.removeAll(ship.positionsOfAllMastInShip());
-    return this;
+  public Set<Position> calculateShipBoundariesPositions(Ship ship) {
+//    boundaries.removeAll(ship.positionsOfAllMastInShip());
```
Comment (rivalez) [inline client/src/main/java/model/ShipBoundariesPositions.java]: :(

File: client/src/main/java/model/Fleet.java (hunk 2 above)
Comment (patiem) [reply client/src/main/java/model/Fleet.java]: I think is better to implement iterable than creating getter which will return list of ships, but if my way of thinking is correct I dont know :)

File: client/src/main/java/gui/playing/PlayBoardController.java (hunk 1 above)
Comment (brrt2) [reply client/src/main/java/gui/playing/PlayBoardController.java]: I don't think it's necessary to create additional variables to use them only to retrieve values from a map

File: client/src/test/java/model/RandomFleetGeneratorTest.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import gui.fields.ClickableField;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class RandomFleetGeneratorTest {
+
+  SoftAssert softAssert = new SoftAssert();
```
Comment (rivalez) [inline client/src/test/java/model/RandomFleetGeneratorTest.java]: private

File: client/src/test/java/model/RandomFleetGeneratorTest.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+  @Test
+  public void returnsFleetWithTenShipsThatAllAreFullyBuilt() {
+
+      //given
```
Comment (rivalez) [inline client/src/test/java/model/RandomFleetGeneratorTest.java]: indentation

File: client/src/main/java/model/RandomFleetGenerator.java (hunk 3 above)
Comment (patiem) [reply client/src/main/java/model/RandomFleetGenerator.java]: Yes, I think this is bad name.

File: client/src/main/java/model/ShipBoundariesPositions.java (hunk 4 above)
Comment (patiem) [reply client/src/main/java/model/ShipBoundariesPositions.java]: mea culpa

File: client/src/test/java/model/RandomFleetGeneratorTest.java (hunk 5 above)
Comment (patiem) [reply client/src/test/java/model/RandomFleetGeneratorTest.java]: ok

File: client/src/test/java/model/RandomFleetGeneratorTest.java (hunk 6 above)
Comment (patiem) [reply client/src/test/java/model/RandomFleetGeneratorTest.java]: ok

**Pull request merged**